File: templates/java/Presenter.java
package de.jochen_schweizer.jsnow.module.##MODULENAME##;


/**
 * Created by ##USERNAME## on ##DATE##.
 */
public class ##MODULENAME##Presenter implements ##MODULENAME##PresenterInput, ##MODULENAME##InteractorOutput{
    private final ##MODULENAME##InteractorInput mInteractor;
    private final ##MODULENAME##PresenterOutput mOutput;

    public ##MODULENAME##Presenter(##MODULENAME##PresenterOutput output, ##MODULENAME##InteractorInput interactor) {
        mOutput = output;
        mInteractor = interactor;
    }
}

File: templates/swift/Presenter.swift
//
//  ##MODULENAME##Presenter.swift
//  JSNow
//
//  Created by ##USERNAME## on ##DATE##.
//  Copyright © 2015 Jochen Schweizer GmbH. All rights reserved.
//

import XCGLogger

class ##MODULENAME##Presenter: NSObject {

    var wireframe: ##MODULENAME##Wireframe!
    weak var userInterface: ##MODULENAME##ViewInterface?
    var interactor: ##MODULENAME##Interactor!

}

// MARK: ##MODULENAME##ModuleInterface

extension ##MODULENAME##Presenter: ##MODULENAME##ModuleInterface {

}

// MARK: ##MODULENAME##InteractorOutput

extension ##MODULENAME##Presenter: ##MODULENAME##InteractorOutput {

}
